src/libs/ReportUtils.ts

```diff
@@ -1235,6 +1235,23 @@ function isMoneyRequestReport(reportOrID: OnyxEntry<Report> | string): boolean {
     return isIOUReport(report) || isExpenseReport(report);
 }
 
+/**
+ * Checks if a report has only one transaction associated with it
+ */
+function isOneTransactionReport(reportID: string): boolean {
+    const reportActions = ReportActionsUtils.getAllReportActions(reportID);
+    return ReportActionsUtils.getOneTransactionThreadReportID(reportActions) !== '0';
+}
+
+/**
+ * Checks if a report is a transaction thread associated with a report that has only one transaction
+ */
+function isOneTransactionThread(reportID: string, parentReportID: string): boolean {
+    const parentReportActions = ReportActionsUtils.getAllReportActions(parentReportID);
+    const transactionThreadReportID = ReportActionsUtils.getOneTransactionThreadReportID(parentReportActions);
+    return reportID === transactionThreadReportID;
+}
```

Comment:
@tgolen we discussed previously about how using methods like `getAllReportActions` isn't the best practice. However, I thought it would be cleaner to implement it like this for now, and make a separate PR to update all the instances of `ReportUtils.getIcons()`, `ReportUtils. shouldReportBeInOptionList()`, and `ReportUtils.shouldReportShowSubscript()` (which use these methods) to send across the appropriate `reportActions` instead of just `reportID`s. 

Lemme know if you disagree and I can add those changes in this PR instead.